### PR TITLE
#165273489 Add article readtime feature

### PR DESCRIPTION
**Pull request by sanya-kenneth**
### What does this PR do?
- Add calculate read time functionality
- Add interprete read time functionality
- Integrate read time calculator with articles app

### How to test this.
- Create an article 
- Check for read_time in the response body

`[POST]  /api/articles/`

`Request body`

```
{
    "title": "Fresh kid wonders on stage at lugogo",
    "description": "he wows the kids",
    "body": "Fresh kid is a 5 year musician who has been on map.",
            "author": "jean",
            "tagList": []
}
```
`Response body`

```
{
    "article": {
        "article": {
            "id": 33,
            "title": "Fresh kid wonders on stage at lugogo",
            "description": "he wows the kids",
            "body": "Fresh kid is a 5 year musician who has been on map.",
            "slug": "fresh-kid-wonders-on-stage-at-lugogo",
            "created_at": "2019-05-09T08:07:40.889971Z",
            "updated_at": "2019-05-09T08:07:40.889999Z",
            "likes_count": 0,
            "dislikes_count": 0,
            "favorited": false,
            "favorite_count": 0,
            "read_time": "less than a minute read",
            "tagList": [],
            "author": "adminee"
        }
    }
}
```

### Relevant PT stories
[#165273489](https://www.pivotaltracker.com/story/show/165273489)

### CODE REVIEW

File: authors/apps/articles/readtime_engine.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    
+    def read_time(self):
+        # minutes in an hour
+        hour = 60
+        # hours a day
+        day = 24
+        # get calculated time
+        time = self.calculate_read_time()
+        # convert minutes into hours
+        minutes = round(time/hour)
```
Comment (joelmugaya) [inline authors/apps/articles/readtime_engine.py]: The minutes variable is not being used

**Review by KabohaJeanMark — APPROVED** [review]
This LGTM. I have tested it out on local and it is working fine. Good work on the logic Sanya.

**Review by joelmugaya — APPROVED** [review]
this is good work

**Review by engjames — APPROVED** [review]
LGTM @sanya-kenneth  nice work

**Review by 3Nakajugo — APPROVED** [review]

File: authors/apps/articles/readtime_engine.py (hunk 1 above)
Comment (sanya-kenneth) [reply authors/apps/articles/readtime_engine.py]: fixed this

**Review by mwinel — APPROVED** [review]
LGTM.

**Review by archibishop — APPROVED** [review]
@sanya-kenneth LGTM i am merging this.

**Pull request merged**